packages/code-du-travail-frontend/src/outils/indemniteLicenciement/StepItems.js

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 import { theme } from "@cdt/ui";
 /**
  * For now, only <Step /> are used but stepItems will
- * allows to navigate between form's steps
+ * allows to navigate between form’s steps
```

Comment:
Is it a problem if we have real apostrophe in comments ?

Reply:
no idea !

Reply:
I  don’t use it since it’s need a combination of keys and I find it not to type compare to apostrophe. Also it’s probably a French typographical specificity since most English comment don’t use typographical quote or apostrophe.

Reply:
The issue is that it's hard to replace them automatically if I must make a distinction between the apostrophe in the text and in comments. But it's not specific to french typography !
(https://en.wikipedia.org/wiki/Apostrophe [typographic form])
```Both simplifications carried over to computer keyboards and the ASCII character set. However, although these are widely used due to their ubiquity and convenience, they are deprecated in contexts where proper typography is important.```

Reply:
But they are not mendatory at all so we can skip this PR I would still manage to sleep 😄

Reply:
>  they are deprecated in contexts where proper typography is important.
JSX => OK
Comment => NOK 

aussi il faudrait plutot voir comment eslint ne pourrait pas le faire à notre place

Reply:
Ah mais complètement ! J'espère qu'il y a une règle pour ça, je vais check

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/327 C'est tout ce que j'ai trouvé. C'est vrai que les false positive peuvent être nombreux. Donc bon, on dirait bien que ça n'existe pas pour le moment et que si un jour c'est mergé dans ce proj, ce sera "juste" un warning.